package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "check-style": "eslint .",
     "pretest": "npm run check-style",
-    "test": "istanbul cover -R _mocha -- --recursive ./test"
+    "test": "istanbul cover -R _mocha -- --recursive ./test",
+    "test-nocov": "mocha --recursive ./test"
```

Comment:
It's that and also some issue with running tests on Windows does to some env var settings.